index.css

```diff
@@ -0,0 +1,225 @@
+html,body {
+    margin: 10px;
+    background-color: #ffeead;
+  }
+  
+      .wrapper {
+          display: grid;
+          grid-gap: 5px;
+          grid-template-columns: repeat(12, [col] 1fr) ;
+          /* grid-template-rows: repeat(6, [row] auto ); */
+          grid-template-rows: auto 150px 150px 150px 150px auto;
+          background-color: #ffeead;
+          color: #444;
+      }
+  
+      .box {
+          padding: 10px;
+          font-size: 150%;
+      }
+  
+      .box .box {
+          font-size: 1em;
+      }
+
+      ul {
+        display: block;
+        list-style-type: none;
+        margin-block-start: 0;
+        margin-block-end: 0;
+        margin-inline-start: 0px;
+        margin-inline-end: 0px;
+        padding-inline-start: 0;
+     }
+
+      nav {
+          flex: 1;
+          display: inline;
+      }
+
+      nav ul {
+          display: flex;
+      }
+
+      nav li {
+        list-style: none;
+      }
+
+      .gallery {
```

Comment:
this is not the best solution. What if there will be added one more link after "gallery"? The layout will break. Better set `margin-left: auto;` to the last child of `ul`, this will position it correctly and will give you more independent layout